west.yml

```diff
@@ -253,7 +253,7 @@ manifest:
       groups:
         - hal
     - name: hal_xtensa
-      revision: a2d658525b16c57bea8dd565f5bd5167e4b9f1ee
+      revision: baa56aa3e119b5aae43d16f9b2d2c8112e052871
```

Comment:
@gbernatxintel  Any connection between this and https://github.com/zephyrproject-rtos/zephyr/pull/78474 ?

Reply:
@kv2019i 
I haven't seen this commit before, but at the moment it is the case that I have 2 commits, 1 in zephyr, the other in sof. When the commit in zephyr is merged I will also have to change in west.yaml in sof and then it can be merged. 

Also if you have time you can have a look and do a review:
https://github.com/zephyrproject-rtos/zephyr/pull/78474
https://github.com/thesofproject/sof/pull/9475